apiserver/facades/client/modelconfig/register.go

```diff
@@ -44,14 +44,15 @@ func makeFacade(stdCtx context.Context, ctx facade.ModelContext) (*ModelConfigAP
 	modelSecretBackend := domainServices.ModelSecretBackend()
 
 	configService := domainServices.Config()
-	modelInfo, err := domainServices.ModelInfo().GetModelInfo(stdCtx)
+	modelSericve := domainServices.ModelInfo()
+	modelInfo, err := modelSericve.GetModelInfo(stdCtx)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 	return NewModelConfigAPI(
 		modelInfo.UUID,
```

Comment:
Would you mind doing a really quick drive by here? We can pull this info off of the context now and remove the `GetModelInfo` call above.